libs/challenge-registry/ui/src/_lib-theme.scss

```diff
@@ -1,9 +1,34 @@
+@use '@angular/material' as mat;
+
+@use './lib/palette' as pal;
 @use './lib/button-github/button-github-theme' as button-github;
 @use './lib/footer/footer-theme' as footer;
 @use './lib/navbar/navbar-theme' as navbar;
 
+@include mat.core();
+@import url('https://fonts.googleapis.com/css2?family=Fira+Sans:wght@300,500&family=Poppins:wght@500,600&display=swap');
```

Comment:
I get the following error in FF:

```console
The resource from “https://fonts.googleapis.com/css2?family=Fira+Sans:wght@300,500&family=Poppins:wght@500&display=swap” was blocked due to MIME type (“text/html”) mismatch (X-Content-Type-Options: nosniff).
```

[Learn more](https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/X-Content-Type-Options)

<img width="542" alt="image" src="https://user-images.githubusercontent.com/3056480/175169140-2231deac-870b-47e6-a1d6-40c8428fa2ab.png">

Do you also have this issue?

Also curious to know whether using `@import` is (still?) the recommended way to import a font (now that we are importing other files with `@use`).

Reply:
Hm, weird!  I have not encountered this error before.

And yes, as mentioned in the footer PR, `@import` is still the method to import a font.  `@use` is to replace Sass' usage of [`@import`](https://sass-lang.com/documentation/at-rules/import), not [CSS'](https://developer.mozilla.org/en-US/docs/Web/CSS/@import).